packages/itwinui-css/src/surface/surface.scss

```diff
@@ -22,5 +22,63 @@
 
 // Public facing surface component.
 @mixin iui-surface-component {
+  --iui-surface-padding-inline: var(--iui-size-s);
+  --iui-surface-content-padding: var(--iui-surface-padding-inline);
+
   @include iui-surface;
+  display: flex;
+  flex-direction: column;
+  height: var(--iui-surface-content-height);
+  width: 100%; // Copying line 16 of `fluid-grid.scss` in iTwinUI-layouts, unsure if needed
```

Comment:
Copied from https://github.com/iTwin/iTwinUI-layouts/blob/10c77cf575d8431d1e4794ba1d48b8aaaa16f634/packages/layouts-css/src/fluid-grid.scss#L16, unsure if needed.

Reply:
Does not belong in this component. fluid-grid will set it if needed

Reply:
Sounds good, removed.